src/iop/basecurve.c

```diff
@@ -1792,22 +1792,15 @@ static gboolean _scrolled(GtkWidget *widget, GdkEventScroll *event, gpointer use
 
   if(c->selected < 0) return TRUE;
 
-  int handled = 0;
-  float dy = 0.0f;
-  if(event->direction == GDK_SCROLL_UP)
-  {
-    handled = 1;
-    dy = BASECURVE_DEFAULT_STEP;
-  }
-  if(event->direction == GDK_SCROLL_DOWN)
+  gdouble delta_x, delta_y;
```

Comment:
>> It feels right, when possible, for touch scrolling to move the point in the direction of the scrolll...

> Yep, same is valid for mouse scroolwheel/etc

I don't have a scroll wheel mouse here to check what is right for it. I should make the scrolling for all cases as it is for touch/trackpad in latest commit, yes?